test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -68,6 +69,7 @@ func InitVolumeExpandTestSuite() TestSuite {
 				testpatterns.BlockVolModeDynamicPV,
 				testpatterns.DefaultFsDynamicPVAllowExpansion,
 				testpatterns.BlockVolModeDynamicPVAllowExpansion,
+				testpatterns.NtfsDynamicPVAllowExpansion,
```

Comment:
I think not all drivers are going to support this yet. Can you add a new test driver capability to gate the test?

Reply:
what is the difference between this one and DefaultFsDynamicPVAllowExpansion or BlockVolModeDynamicPVAllowExpansion?

Reply:
I updated to use NtfsDynamicPV. I think this way similar to DefaultFsDynamicPV, if driver has the expansion capability, it will trigger the tests. PTAL

Reply:
What if the driver has not been updated to support expansion on ntfs yet?

Reply:
actually this one can only trigger negative test 
volume-expand should not allow expansion of pvcs without AllowVolumeExpansion property

I was thinkning what is the difference between ntfs and defaultfs, here we have DefaultfsDynamicPVAllowExpansion, does it mean all driver which use defaultFs can support allowexpansion?

Reply:
So I think my previous setup seems ok. And actually I should have both NtfsDynamicPVAllowExpansion and NtfsDynamicPV. 
THis way, for driver which has capability of controllerExpansion, it will test positive tests, for drivers which do not has this capablility, it will test negative test.

Reply:
Yes but what if you have a driver that supports volumeexpansion on linux, but not yet on windows?

Reply:
I guess you would need two different test configs? one for linux and one for windows

Reply:
yes, we have different setting for linux and windows. now since both support volume resize, both have the capability set.
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/12ea362bb378bb52fbbbb064407889719092ffea/test/k8s-integration/driver-config.go#L60